src/test/java/org/rulez/demokracia/pdengine/ContextTest.java

```diff
@@ -0,0 +1,37 @@
+package org.rulez.demokracia.pdengine;
+
+import static org.junit.Assert.*;
+import static org.mockito.Mockito.*;
+
+import javax.xml.ws.WebServiceContext;
+
+import org.hibernate.Session;
+import org.hibernate.SessionFactory;
+import org.junit.Before;
+import org.junit.Test;
+import org.rulez.demokracia.pdengine.annotations.tested_behaviour;
+import org.rulez.demokracia.pdengine.annotations.tested_feature;
+import org.rulez.demokracia.pdengine.annotations.tested_operation;
+import org.rulez.demokracia.pdengine.exception.ReportedException;
+import org.rulez.demokracia.pdengine.testhelpers.CreatedDefaultVoteRegistry;
+
+public class ContextTest extends CreatedDefaultVoteRegistry {
+
+
+	@Before
+	public void setUp() throws ReportedException {
+		super.setUp();
+	}
+	
+	@tested_feature("Supporting functionality")
+	@tested_operation("get context parameter")
+	@tested_behaviour("context parameters can be obtained from the VoteManager")
+	@Test
+	public void testContextParameterCanBeObtainedFromVoteManager() {
```

Comment:
The test case name is directly going to the documentation, so it is important to have proper sentences here.
See the generated doc to understand how does it work: https://adadocs.demokracia.rulez.org/PDEngine/edemo/581/engine-implementedBehaviours.html#context%20parameters%20can%20be%20obtained%20from%20the%20VoteManager
public void getWsContext_returns_the_webservice_context() would be a good name here.